### ISSUE TITLE
Husky dependency causes issues when building app

### ISSUE THREAD

**Issue by dakotabuck** [dromru/react-photoswipe-gallery#472]
Hi, I encountered a build issue. It seems to be a problem with Husky. I tested this on both Linux and Windows.

**Steps to reproduce:**

- Begin new React project
- Install react-photoswipe gallery `npm install photoswipe react-photoswipe-gallery --save`
- Run project via any means e.g. `npm run dev` or `npm run build`

**Expected Behavior:** build succeeds, application server runs with react-photoswipe-gallery

**Observed Behavior:** build fails with the following error:

       -----> Installing dependencies

       Installing node modules

       

       > core-js@2.6.12 postinstall /tmp/build_0f71b228/node_modules/core-js

       > node -e "try{require('./postinstall')}catch(e){}"

       

       

       > react-photoswipe-gallery@1.3.5 postinstall /tmp/build_0f71b228/node_modules/react-photoswipe-gallery

       > husky install

       

       sh: 1: husky: not found

       npm ERR! code ELIFECYCLE

       npm ERR! syscall spawn

       npm ERR! file sh

       npm ERR! errno ENOENT

       npm ERR! react-photoswipe-gallery@1.3.5 postinstall: `husky install`

       npm ERR! spawn ENOENT

       npm ERR! 

       npm ERR! Failed at the react-photoswipe-gallery@1.3.5 postinstall script.

       npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

**Notes:** I was able to work around this temporarily by editing the `index.js` file this error references to continue execution without existence of `.git` folder, but I don't know if this is a problem with `react-photoswipe-gallery` or with its dependency `husky`.

**Comment by irudoy** [comment]
@dakotabuck thanks for reporting!

**Comment by BradNut** [comment]
This is an issue again on the newest change for 4.1.0:
```
✔ Run npm install to install new versions? … yes
Installing dependencies...
npm error code 127

npm error path /home/username/project/node_modules/react-photoswipe-gallery
npm error command failed
npm error command sh -c husky install
npm error sh: 1: husky: not found

npm error A complete log of this run can be found in: /home/username/.npm/_logs/2026-06-14T19_07_04_909Z-debug-0.log
```

**Comment by akhmadullin** [comment]
@BradNut, thank you for reporting.

Fixed it in [4.1.2](https://www.npmjs.com/package/react-photoswipe-gallery/v/4.1.2).
